packages/Tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+
+import cx from 'classnames';
+
+import CrownSolid from '@igloo-ui/icons/dist/CrownSolid';
+import TextBulletSolid from '@igloo-ui/icons/dist/TextBulletSolid';
+
+import './tabs.scss';
+
+export interface TabInterface {
+  /** The tab pane contents of the tabs list */
+  children?: React.ReactNode;
+  /** Add an id attribute to the tab */
+  id?: string;
+  /** Add text for the tab */
+  label: React.ReactNode;
+  /** Add a notification indicator to the tab */
+  notification?: boolean;
+  /** Add a premium indicator to the tab */
```

Comment:
In you documentation, you should decide if you write `Tab` or `tab` to have continuity between each other.

Reply:
I decided to use lowercase T since it's proper English that way..